build.gradle

```diff
@@ -134,3 +134,109 @@ def getProjectVersion() {
     }
     return ver
 }
+
+apply plugin: 'distribution'
+
+task sourceCopy(type: Copy) {
+    from rootDir
+    into 'source'
+}
+
+task sourceTar(type: Tar) {
+    dependsOn 'sourceCopy'
+    from  'source'
+    destinationDir = file('sourceArtifacts')
+    extension = 'tgz'
+    compression = Compression.GZIP
+}
+
+task javadocs(type: Javadoc) {
+    description = "Generate main pravega javadoc"
+}
+
+task publishAllJars() {
```

Comment:
This should invoke the publish task for schema-registry. This is usually used for publishing the release artifacts or snapshot versions. (publishUrl is read in the maven.gradle file).

Reply:
removed the placeholder